maint_tools/check_gha_workflow.py

```diff
@@ -61,13 +61,9 @@
 # file containing your github token
 TOKEN_FILE = Path("/home/remi/Documents/tokens/gh_read_repo_for_orga.txt")
 
-# can be found out at
-# "https://api.github.com/repos/{USER}/{REPO}/actions/workflows"
-WORKFLOW_ID = "71549417"
-
 BRANCH = "main"
 
-INCLUDE_FAILED_RUNS = True
+INCLUDE_FAILED_RUNS = False
```

Comment:
I'm wondering what this entails... Can you explain the rationale ?

Reply:
Will add some doc to explain this.

Reply:
Added some explanation.
@bthirion is this clearer?

Reply:
OK